### PR TITLE
Complete README example for node client

### PR DESCRIPTION
**Pull request by jmilet**
## Description
This PR completes the sample code included in the README for the node client.

## Related Issue
https://app.asana.com/0/1200164532266868/1200233999087172

## Motivation and Context
This PR is part of the Mazzaroth project.

## How Has This Been Tested?
Unit test.

## Screenshots (if appropriate):
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,74 +67,257 @@ For a more detailed example of how to use the contract client in conjunction
...
+
+```bash
+mazzaroth-cli deploy deploy.json
+```
+
 ### Node-Client
 
 ```js
 import { NodeClient } from 'mazzaroth-js'
+import { fromPrivate } from 'mazzaroth-js/dist/crypto/ecc-ed25519'
```
Comment (jeff1010322) [inline README.md:88]: You can change this to use the public function:
AddressFromPrivate

https://github.com/kochavalabs/mazzaroth-js/blob/76e368b3e89c88ca7933acdee2094de7198943d6/src/index.js#L19

File: README.md [hunk 2]
```diff
@@ -67,74 +67,257 @@ For a more detailed example of how to use the contract client in conjunction
...
+
+async function demo() {
+  // Private key for the account is 0x64.
+  const accountPrivKey = '0'.repeat(64)
+  const mazzNodeAddr = 'http://localhost:8081'
+  const accountID = fromPrivate(accountPrivKey).toString('hex')
+  const channelID = '0'.repeat(64)
+
+  // Crate a client.
```
[the remark this answers is not in the document]
Comment (jmilet) [reply README.md]: Done. Thanks.

File: README.md (hunk 1 above)
Comment (jmilet) [reply README.md]: Done. Thanks.

File: README.md [hunk 3]
```diff
@@ -67,74 +67,257 @@ For a more detailed example of how to use the contract client in conjunction
...
-      statusInfo: 'Transaction has been accepted and is being executed.' }
-  **/
-})
-```
+      statusInfo: 'Readonly request executed successfully.'
+    }
+  */
+}
+
+demo()```
```
[the remark this answers is not in the document]
Comment (jmilet) [reply README.md]: Done. Thanks.

**Review by jeff1010322 — APPROVED** [review]

**Pull request merged**